cmd/export.go

```diff
@@ -155,22 +169,23 @@ func (o *ExportOptions) Run() error {
 	}
 	path = filepath.Join(path, dsr.Name)
 
-	// TODO: Implement flags specified in the RFC 0014, only set the zip flag if the export
-	// includes multiple files and the --directories-for-files flag is not true.
-	o.Zipped = true
-	if o.Zipped {
-		dst, err := os.Create(fmt.Sprintf("%s.zip", path))
-		if err != nil {
-			return err
-		}
+	// TODO (dlong): When --zipped flag is supported, don't always do this.
+	dst, err := os.Create(fmt.Sprintf("%s.zip", path))
+	if err != nil {
+		return err
+	}
 
-		if err = dsutil.WriteZipArchive(o.Repo.Store(), ds, res.String(), dst); err != nil {
-			return err
-		}
-		return dst.Close()
+	// TODO (dlong): Use --body-format here to convert the body and ds.Structure.Format, before
+	// passing ds to WriteZipArchive.
+	if err = dsutil.WriteZipArchive(o.Repo.Store(), ds, format, res.String(), dst); err != nil {
+		return err
 	}
+	return dst.Close()
 
-	if path != "" {
+	// TODO (dlong): Document the full functionality of export, and restore this code below. Allow
```

Comment:
I think we should elevate this to an issue and block the next release until it's shipped. Getting export in a better place has jumped up our list of priorities. Mind filing an issue so others can see what's going on there? No need to block merging tho

Reply:
Agreed, will do.